### PR TITLE
More tests for pandas and metrics

### PR DESCRIPTION
**Pull request by marta-sd**

### CODE REVIEW

File: tests/test_pandas.py [hunk 1]
```diff
@@ -130,6 +130,49 @@ def test_csv():
     assert_equal(df.columns.tolist(), df2.columns.tolist())
     assert_equal(df.dtypes.tolist(), df2.dtypes.tolist())
 
+    with NamedTemporaryFile(suffix='.csv', mode='w+') as f:
+        df.to_csv(f, index=False, columns=['name', 'act'])
```
Comment (mwojcikowski) [inline tests/test_pandas.py]: Pandas 0.13 uses `cols` kwarg. https://pandas.pydata.org/pandas-docs/version/0.13/generated/pandas.DataFrame.to_csv.html We need to fix that bug.

File: tests/test_pandas.py [hunk 2]
```diff
@@ -130,6 +130,49 @@ def test_csv():
...
+
+
+def test_excel():
+    # just check if it doesn't fail
+    df = opd.read_sdf(os.path.join(test_data_dir, 'data/dude/xiap/actives_docked.sdf'),
+                      columns=['mol', 'name', 'chembl_id', 'dude_smiles', 'act'])
+    df['act'] = df['act'].astype(float)
+    df['name'] = df['name'].astype(int)
+    with NamedTemporaryFile(suffix='.xls', mode='w') as f:
+        df.to_excel(f.name, index=False)
```
Comment (mwojcikowski) [inline tests/test_pandas.py]: All this methods should work with `path_or_buf` so `f` should work and tested. If `path_or_buf` is empty, `str` should be returned, thus might be done in one line (altogether 3 tests per 'writer').

File: tests/test_metrics.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+from nose.tools import (assert_less_equal, assert_less, assert_greater_equal,
+                        assert_greater,
+                        assert_equal)
+
+import numpy as np
+np.random.seed(42)
```
Comment (mwojcikowski) [inline tests/test_metrics.py]: Imports first

Comment (marta-sd) [reply tests/test_metrics.py]: I know the rule, but I always ignore it when it comes to random seeds. I had bad experience with setting the seed after all imports (other module might use it at import stage)

File: tests/test_pandas.py (hunk 2 above)
Comment (marta-sd) [reply tests/test_pandas.py]: We can (and should) do such tests for CSV writer, but API for Excel is different and only file name or ExcelWriter object are supported (see [docs](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.to_excel.html)) Btw we currently don't support ExcelWriter, but it's an easy fix

File: tests/test_metrics.py (hunk 3 above)
Comment (mwojcikowski) [reply tests/test_metrics.py]: I prefer setting seed right where using them, so locally, not in a global context. Whichever you prefer is also fine by me.

Comment (marta-sd) [reply tests/test_metrics.py]: Let's set them locally then

File: tests/test_pandas.py (hunk 1 above)
Comment (marta-sd) [reply tests/test_pandas.py]: Fun fact: it uses `columns` in `to_html`: https://pandas.pydata.org/pandas-docs/version/0.13/generated/pandas.DataFrame.to_html.html#pandas.DataFrame.to_html

**Pull request merged**